tool/tctl/common/auth_command.go

```diff
@@ -289,7 +289,14 @@ func (a *AuthCommand) ExportAuthorities(ctx context.Context, clt authCommandClie
 	}
 
 	// Only a single CA is exported if we got this far.
-	fmt.Printf("%s\n", authorities[0].Data)
+	data := authorities[0].Data
+
+	// Windows exports are DER-encoded.
+	if a.authType == "windows" {
```

Comment:
This is a rather crude check - it would be better if `authorities[0]` had a field that indicated what format the data is in.

@codingllama WDYT?

Reply:
Yep, that's what the struct is there for.

Printing binary data straight to the terminal does seem a bit iffy, but I suppose it's in keeping with legacy behavior. (Also it could be redirected.)

Reply:
So should we add a field to the struct indicating the data type?

Reply:
Yes, seems reasonable to me.